docs/products/kafka/kafka-connect/howto/manage-connector-versions.md

```diff
@@ -21,7 +21,7 @@ service configuration.
   upgrade. The connector will continue to run, but upgrading to a supported version is
   recommended to avoid compatibility issues.
 - Support is limited to the latest connector version and the most recent previous
-  version. Breaking changes, if any, are detailed in
+  version. Breaking changes, if known, are mentioned in
```

Comment:
“if known” might suggest that some breaking changes are unknown and not mentioned.  I think we can keep the previous line as is? Also, “are detailed” uses a stronger verb than “are mentioned.”